### PR TITLE
Cleanup handling of tests for update queries

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by cberner** [comment]
I think you broke the tests (see Travis)

File: presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java [hunk 1]
```diff
@@ -75,18 +77,28 @@
     private final List<Type> types;
     private final Map<String, String> setSessionProperties;
     private final Set<String> resetSessionProperties;
+    private final Optional<String> updateType;
+    private final OptionalLong updateCount;
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java]: Should these be part of `equals` and `hashCode`

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java [hunk 2]
```diff
@@ -123,19 +124,39 @@ protected void assertQueryOrdered(@Language("SQL") String actual, @Language("SQL
...
-        assertQueryTrue(getSession(), sql);
+        assertUpdate(getSession(), actual, expected);
     }
 
-    protected void assertQueryTrue(Session session, @Language("SQL") String sql)
+    protected void assertUpdate(Session session, @Language("SQL") String actual, @Language("SQL") String expected)
             throws Exception
     {
-        assertQuery(session, sql, "SELECT true");
+        QueryAssertions.assertQuery(queryRunner, session, actual, h2QueryRunner, expected, false, true);
```
Comment (dain) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java]: This should be updated to compare results

**Comment by dain** [comment]
Minor comments... looks good

**Pull request closed without merging**